apps/wallet-mobile/src/components/AmountItem/AmountItem.tsx

```diff
@@ -8,7 +8,7 @@ import {isEmptyString} from '../../utils'
 import {YoroiWallet} from '../../yoroi-wallets/cardano/types'
 import {useTokenInfo} from '../../yoroi-wallets/hooks'
 import {Quantities} from '../../yoroi-wallets/utils'
-import {Boundary, Icon, Placeholder, Spacer, Text, TokenIcon} from '..'
+import {Boundary, Icon, Spacer, Text, TokenIcon, TokenIconPlaceholder} from '..'
```

Comment:
Skeletons?

Reply:
What do you mean? The name of the Placeholder/Skeleton?

TokenIcon was exporting Placeholder to the global component scope, which was very generic, so I renamed it TokenIconPlaceholder. And created a different one for the AmountItemPlaceholder

Called them placeholders instead of skeletons because that's the name it had before

Reply:
I think `TokenIconPlaceholder` is used to replace the original image/logo when not present, `Amount` looks like is used only as fallback's boundary so I'd say he is the candidate.